.circleci/config.yml

```diff
@@ -205,6 +205,7 @@ workflows:
       - run_tests
       - test_installation_from_source_test_mode
       - test_installation_from_source_develop_mode
+      - build_documentation
```

Comment:
Could you remove this line again please?

Reply:
ah crap! yes